### ISSUE TITLE
Currency page error after upgrade to 1.7.7.2

### ISSUE THREAD

**Issue by darkf3d3** [PrestaShop/PrestaShop#23800]
After upgrading from PS1.7.4.2 to PS1.7.7.2 using last version on 1click upgrade module, I'm no more able to edit currencies from currencies page:
.../admin/index.php/improve/international/currencies/
The page loaded correctly, but when i click on the pencil icon next to one currency, this error page it's showed:
![Immagine 2021-03-29 003711](https://user-images.githubusercontent.com/42684711/112770564-7e2dff00-9027-11eb-9a7a-19cbda52ca43.png)

**Comment by hibatallahAouadni** [comment]
Hello @darfk3d3 

Thanks for your issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must follow the template to submit a complete bug report**.

Read more about how we expect the issues to be handled [here](https://build.prestashop.com/news/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it.

PS: it seems your issue similar to this one #22177, could you please check it :slightly_smiling_face: 

Thanks!

**Comment by darkf3d3** [comment]
Hi @hibatallahAouadni
that issue it's not like the one I'm encountering.
In my case there's no way to change currency settings because currency page show the error posted above.

To reproduce the error, after upgrading from PS1.7.4.2 to PS 1.7.7.2
1. From Back office left menu
2. Go to International >> Tax
3. Click on the pencil icon
4. At this point the error it's showed.

**Comment by hibatallahAouadni** [comment]
Hello @darfk3d3 

I didn't manage to repropduce your issue following your steps, see the attached screen record below:

 https://www.awesomescreenshot.com/video/3220807?key=6f1486087b36340ddaafdecf7f8cf489

Thanks!

**Comment by darkf3d3** [comment]
Hello @hibatallahAouadni hibatallahAouadni 
I'm here with some more informations.
Previously I found the error on currency page, but i noticed that setting if i set the current employe language to English every back office page stop working, showing this error:
![errore](https://user-images.githubusercontent.com/42684711/113313631-b7a68900-930b-11eb-9244-68a0deedf1a5.png)

I always used the account in Italian, so I didn't noticed this as first.
But I just tested the version before the upgrade (1.6.0.9) and it works on all languages.
And after the upgrade to PS1.7.7.3 and set the employe language from italian to english, the above error it's showed.
Maybe this can help replicate the problem?
